jmx-scraper/src/main/java/io/opentelemetry/contrib/jmxscraper/config/JmxScraperConfig.java

```diff
@@ -55,9 +55,9 @@ public class JmxScraperConfig {
   private String serviceUrl = "";
   private String customJmxScrapingConfigPath = "";
   private Set<String> targetSystems = Collections.emptySet();
-  private int intervalMilliseconds;
-  private String metricsExporterType = "";
-  private String otlpExporterEndpoint = "";
+  private int intervalMilliseconds; // TODO only used to set 'otel.metric.export.interval' from SDK
```

Comment:
What this TODO means here? Is there anything aditional to be implemented?

Reply:
Yes, basically this configuration option is only used to configure the metrics polling interval and there is already a dedicated configuration option in the SDK for this, so if someone is used to configure the SDK using this one we should reuse the existing config option (we'll deal with that when adding auto-configuration here).